Sources/Clappr_iOS/Classes/Plugin/Core/MediaControl/TimeIndicator.swift

```diff
@@ -76,12 +75,13 @@ open class TimeIndicator: MediaControlPlugin {
     }
 
     private func bindEvents() {
+        stopListening()
         bindContainerEvents()
         bindPlaybackEvents()
         bindCoreEvents()
     }
 
-    private func bindContainerEvents() {
+    open func bindContainerEvents() {
```

Comment:
While reviewing, this change made me think about the "openess" of Clappr plugins. Maybe it's a discussion that is worth a small sync meeting. The question is: what will be the philosophy of all the built-in plugins in Clappr? Are we going to `open` all the methods from those plugins? 

I took a quick look at other plugins, and we don't have a standard on such methods. Some of them have private `bindXEvents` methods but some of them have half `open` and half `private`. I believe we should establish a standard on plugin methods. WDYT?

Reply:
@jademcosta this makes total sense.

I believe we can open another PR exposing the methods of our `MediaControl` plugins. That way every user can inherit those plugins if needed.

What do you think?

Reply:
Makes sense :)